Src/Delivery/studio/templates/code/array_template.e

```diff
@@ -0,0 +1,32 @@
+class ARRAY_TEMPLATE [T -> COMPARABLE]
```

Comment:
Not required but I think it would be nice to have a note clause at the top with our usual tags: description, license, date, revision.